packages/itwin/grouping-mapping-widget/src/test/QueryBuilder.testdata.ts

```diff
@@ -8,7 +8,7 @@ export const testCases: QueryBuilderTestData = {
   testCases: [
     {
       name: "when non-relational property added, return query string with property value",
-      expectedResult: "SELECT A.B.ECInstanceId FROM A.B WHERE A.B.somePropName=1",
+      expectedResult: "SELECT A.B.ECInstanceId FROM A.B WHERE A.B.somePropName = 1",
```

Comment:
Why were the spaces added?

Reply:
To improve consistency as class joins have spaces, but properties don't.
And from what I read online, it is a recommended guideline to have spaces (auto formatters do this as well)